e2e/tests/components/input-dropdown/input-dropdown.e2e-spec.ts

```diff
@@ -0,0 +1,32 @@
+import { imageCompare } from '../common/image-compare';
+import { InputDropdownPage } from './input-dropdown.po.spec';
+
+describe('Input Dropdown Tests', () => {
+
+    let page: InputDropdownPage;
+
+    beforeEach(async () => {
+        page = new InputDropdownPage();
+        await page.getPage();
+    });
+
+    it('should have correct initial states', async () => {
+
+        // dropdown list not expanded
+        expect(await page.confirmDropdownIsExpanded()).toBeFalsy();
+
+        expect(await page.getText()).toEqual('Selection: (none)');
+
+        expect(await imageCompare('input-dropdown-initial')).toEqual(0);
+
+    });
+
+    it('should disable input dropdown when the disabled button is checked', async () => {
+
+        await page.checkboxDisabled.click();
+
+        expect(await imageCompare('input-dropdown-disabled')).toEqual(0);
+
+    });
+
```

Comment:
We should also have a test where the control is disabled via the reactive forms API. To do this we would have one input dropdown that is standalone and one that is in a form. The select component has tests like this, for reference.